.build/.kevm.rev

```diff
@@ -1 +1 @@
-5d3bbb6d9dd3a2913201071937cb3f00d66832b0
+765edc205dd122242c31739372d90c1fcc38a9c3
```

Comment:
first, you should use this KEVM version:
https://github.com/kframework/evm-semantics/commits/e2492651b03e8d7b0b0789592ad7bac96a72ef5b

forget about `keq` branches from now on. it's the temporary version available when you started this work. now you should use the above KEVM version. please make sure the current spec works with the version.  also, if you made any changes to the KEVM for this spec, let me know.

Reply:
second, you should not change this global .build directory, which affects all other specs. you should create a new .build directory to which zeppelin and vyper Makefiles refer.